central/cluster/datastore/telemetry_test.go

```diff
@@ -0,0 +1,62 @@
+package datastore
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome/telemeter"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome/telemeter/mocks"
+	"go.uber.org/mock/gomock"
+)
+
+func Test_sendProps(t *testing.T) {
+	teleMock := mocks.NewMockTelemeter(gomock.NewController(t))
+	captureTrack := func(id string, props map[string]any) *gomock.Call {
+		return teleMock.EXPECT().Track("Updated Secured Cluster Identity",
```

Comment:
Could you add a comment why we need this 
- https://github.com/golang/mock/issues/68

Reply:
Code removed.